letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -172,11 +174,18 @@ BootstrapDebCommon() {
   # - Debian 6.0.10 "squeeze" (x64)
 
   $SUDO apt-get update || echo apt-get update hit problems but continuing anyway...
+  
+  # checking if certbot/letsencrypt is already packaged with OS
+  if [ "$QUIET" != 1 ]; then
+      if dpkg --get-selections | grep -v letsencrypt.sh | grep -c letsencrypt -gt 0; then
```

Comment:
I don't think we want to check for `letsencrypt` packages; although they're present on Xenial, they're increasingly old and crufty and we want to deprecate them.  Let's just check for `certbot` packages.